snc.sh

```diff
@@ -201,6 +201,10 @@ retry ${OC} patch clusterversion version --type json -p "$(cat cvo-overrides-aft
 # Scale route deployment from 2 to 1
 retry ${OC} scale --replicas=1 ingresscontroller/default -n openshift-ingress-operator
 
+# Set managementState Image Registry Operator configuration from Removed to Managed
+# because https://docs.openshift.com/container-platform/4.15/registry/configuring_registry_storage/configuring-registry-storage-baremetal.html#registry-removed_configuring-registry-storage-baremetal
```

Comment:
This by default doesn't enable the registry but change the state of registry operator from remove to managed. Internal registry default enabled as part of capability what we want is to expose and that have separate doc https://docs.openshift.com/container-platform/4.15/registry/securing-exposing-registry.html

Reply:
We must change it, so it has a user-visible effect.
Documenting in the comment what effect this has on the cluster is more useful than paraphrasing `retry ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"managementState":"Managed"}}' --type=merge` in English, and then expecting the reader to follow a link, and then guess what was achieved with `Removed` -> `Managed` by reading a large block of text.

If "Enable internal registry" is not what this does, then this can be "Re-enable the registry operator so that the internal registry is exposed externally" or whatever this is doing.